<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RFfile</class>
 <widget class="QMainWindow" name="RFfile">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>480</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>RFfile</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/RFfile_icon.qrc">
    <normaloff>:/logo/logo.png</normaloff>:/logo/logo.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QScrollArea" name="scrollArea">
        <property name="font">
         <font>
          <family>Adobe Arabic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>490</width>
           <height>413</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QFrame" name="frame">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="2" column="0">
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="3">
                 <widget class="QPushButton" name="btn_old_file">
                  <property name="text">
                   <string>打开</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLineEdit" name="let_oldfilename">
                  <property name="font">
                   <font>
                    <family>Adobe 黑体 Std R</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>旧的文件</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="1" column="1">
                 <widget class="QLineEdit" name="let_newfilename">
                  <property name="font">
                   <font>
                    <family>Adobe 黑体 Std R</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QPushButton" name="btn_new_file">
                  <property name="text">
                   <string>打开</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>最终文件</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QWidget" name="widget" native="true">
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>镜头表格</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="btn_excel_import">
                  <property name="text">
                   <string>导入</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="let_excel">
                  <property name="font">
                   <font>
                    <family>Adobe 黑体 Std R</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QWidget" name="widget_5" native="true">
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="1" column="0">
                 <widget class="QPushButton" name="btn_reference">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>替换</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QWidget" name="widget_4" native="true">
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="2" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>工作目录</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" alignment="Qt::AlignHCenter">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>模块</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="cbb_displaymod">
                  <property name="font">
                   <font>
                    <family>Adobe 黑体 Std R</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>ArrowCursor</cursorShape>
                  </property>
                  <property name="mouseTracking">
                   <bool>false</bool>
                  </property>
                  <property name="tabletTracking">
                   <bool>false</bool>
                  </property>
                  <item>
                   <property name="text">
                    <string>ani</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>cam ani</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ccp</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>cmp</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>layout</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>lgt</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>mtp</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>pzv</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>reference</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>trk</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>vfx</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QComboBox" name="cbb_displaydir">
                  <property name="font">
                   <font>
                    <family>Adobe 黑体 Std R</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <item>
                   <property name="text">
                    <string>work</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>check</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>approve</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="let_path">
                  <property name="font">
                   <font>
                    <family>Adobe 黑体 Std R</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>工程目录</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QPushButton" name="btn_open_file_path">
                  <property name="text">
                   <string>打开</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTextBrowser" name="tbr_filelist">
            <property name="font">
             <font>
              <family>Adobe 黑体 Std R</family>
              <pointsize>9</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(255, 0, 0);</string>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Adobe 黑体 Std R'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../icons/RFfile_icon.qrc"/>
 </resources>
 <connections/>
</ui>
